src/Command/ImportCitizenProjectSkillsCommand.php

```diff
@@ -0,0 +1,126 @@
+        'Éducateur et animateur spécialisé',
+        'Professeur de sport',
+        'Parent d’élèves',
+        'Traducteur et linguiste',
+        'Artiste',
+        'Professionnel de la culture',
+        'Architecte et designer',
+        'Dessinateur, peintre et sculpteur',
+        'Musicien',
+        'Acteur, professionnel du théâtre et du cirque',
+        'Photographe',
+        'Professionnel des ressources humaines',
+        'Professionnel de la réinsertion professionnelle',
+        'Professionnel de la formation',
+        'Professionnel de la réorientation professionnelle',
+        'Responsable associatif',
+        'Solidarité intergénérationnel',
+        'Lutte contre l’exclusion',
+        'Egalité des chances',
+        'Lutte contre les discriminations femmes-hommes',
+        'Lutte contre les discriminations LGBTQ+',
+        'Handicap',
+        'Migrant, demandeur d’asile et réfugiés',
+        'Entrepreneur',
+        'Développeur d’application',
+        'Développeur de site internet',
+        'Gestion de base de données',
+        'Utilisation des outils bureautiques',
+        'Spécialiste en robotique',
+        'Spécialiste en intelligence artificielle',
+        'Animateur de fablab',
+        'Graphiste 3D',
+        'Gaspillage alimentaire',
+        'Énergies renouvelables',
+        'Recyclage et gestion des déchets',
+        'Agriculteur',
+        'Jardinier et paysagiste',
+        'Isolation thermique et acoustique',
+        'Médecin',
+        'Infirmier(e)',
+        'Aide-soignant(e)',
+        'Psychologue',
+        'Secouriste',
+        'Vétérinaire',
+        'Artisan',
+        'Construction et chantier',
+        'Chef cuisinier',
+        'Restaurateur et hôtelier',
+        'Guide touristique',
+    ];
+
+    private $citizenProjectSkillsRepository;
+    private $entityManager;
+
+    protected function configure()
+    {
+        $this
+            ->setName('app:import:citizen-project-skills')
+            ->setDescription('Import a pre defined list of CitizenProjectSkill')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln('Start import...');
+        $countToAdd = 0;
+        foreach (self::CITIZEN_PROJECT_SKILLS as $skillName) {
+            if ($citizenProjectSkill = $this->citizenProjectSkillsRepository->findOneByName($skillName)) {
+                continue;
+            }
+
+            $citizenProjectSkill = new CitizenProjectSkill($skillName);
+            $this->entityManager->getManager()->persist($citizenProjectSkill);
+            ++$countToAdd;
+        }
+
+        $this->entityManager->getManager()->flush();
+
+        $output->writeln(sprintf('Finish : %s citizenProjectSkills was added', $countToAdd));
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->entityManager = $this->getContainer()->get('doctrine');
```

Comment:
$this->entityManager = $this->getContainer()->get('doctrine.orm.entity_manager');